django/db/models/fields/related_descriptors.py

```diff
@@ -110,7 +110,8 @@ def get_queryset(self, **hints):
             if not getattr(related_model._default_manager, 'silence_use_for_related_fields_deprecation', False):
                 warnings.warn(
                     "use_for_related_fields is deprecated, instead "
-                    "set Meta.base_manager_name on '{}'.".format(related_model._meta.label),
+                    "set Meta.base_manager_name on '{}' then remove "
```

Comment:
They can. The exact semantics are as follows:

* Legacy way alone => old behavior, deprecation warning.
* Mixing legacy and new way => mixed behavior, deprecation warning. *This is the point of this PR, as new way is not fully backwards compatible with old way and updating the API is too late for release 1.10*.
* New way alone => new behavior, no warning

I thought the warning would allow expliciting that new behavior is not fully active until `use_for_related_fields` is not removed. You have the final word anyway, if you prefer that I drop the "then remove use_for_related_fields", I can do so.